Source/Charts/Renderers/RadarChartRenderer.swift

```diff
@@ -37,7 +52,21 @@ open class RadarChartRenderer: LineRadarRenderer
         if radarData != nil
         {
             let mostEntries = radarData?.maxEntryCountSet?.entryCount ?? 0
-            
+
+            // If we redraw the data, remove and repopulate accessible elements to update label values and frames
+            self.accessibleChartElements.removeAll()
+
+            // Make the chart header the first element in the accessible elements array
+            if let chartDescriptionText: String = chart.chartDescription?.text {
```

Comment:
Why does this chart not say the datasets in the chart description is present?

Reply:
I believe this was because, for Radar chart, each individual dataSet is a heading element, so it's the next item in VO scroll order. Can definitely be added here, but I thought it was redundant, so let me know if you think I should add it.